src/excmds.ts

```diff
@@ -5230,8 +5230,15 @@ export async function keyfeed(mapstr: string) {
 //#background_helper
 browser.runtime.onInstalled.addListener(details => {
     if (details.reason === "install") tutor("newtab")
-    else if ((details as any).temporary !== true && details.reason === "update") updatenative(false)
-    // could add elif "update" and show a changelog. Hide it behind a setting to make it less annoying?
+    else if (details.reason === "update") {
+        if ((details as any).temporary !== true) {
+            updatenative(false)
+        } else {
+            // Temporary extension has been updated in place
+            // Reload all pages so Tridactyl can work in them
+            reloadall()
```

Comment:
@Rummskartoffel

The idea of this PR is to make the pages reload (which I agree may be of debatable value - it might be better to run `tutor()` instead).

The problem is that `web-ext` reloads _the extension_ too soon and ends up with stale extension code. Presumably that is independent of this PR.

Reply:
I hadn't noticed that issue before, either. Do full builds have this problem as well?

Reply:
No, they don't. (I said this to you on Matrix but I should probably put it here for posterity)